internal/handlers/common/aggregations/stages/projection/projection.go

```diff
@@ -242,6 +274,7 @@ func ProjectDocument(doc, projection *types.Document, inclusion bool) (*types.Do
 				return nil, err
 			}
 
+			set = true
```

Comment:
This is no covered by tests, I think projection doesn't have any operator tests. Also needs test for recursive operator tests for $project to go with these line changes.

Reply:
Now covered